webapp/src/containers/Rhs/index.tsx

```diff
@@ -47,24 +53,38 @@ const Rhs = (): JSX.Element => {
         return {isLoading, isSuccess, isError, data: data as SubscriptionData[], error: (apiErr as FetchBaseQueryError)?.data as APIError | undefined};
     };
 
-    // Fetch all the subscriptions from the API
+    // Reset the pagination params and empty the subscription list
+    const resetStates = useCallback(() => {
+        setPaginationQueryParams(() => ({page: Constants.DefaultPage, per_page: Constants.DefaultPageSize}));
```

Comment:
```suggestion
        setPaginationQueryParams({page: Constants.DefaultPage, per_page: Constants.DefaultPageSize});
```
What's the need of providing a function here?